flink-runtime/src/test/java/org/apache/flink/runtime/metrics/MetricRegistryConfigurationTest.java

```diff
@@ -38,13 +40,13 @@
 	/**
 	 * TestReporter1 class only for type differentiation.
 	 */
-	private static class TestReporter1 extends TestReporter {
+	static class TestReporter1 extends TestReporter {
 	}
 
 	/**
 	 * TestReporter2 class only for type differentiation.
 	 */
-	private static class TestReporter2 extends TestReporter {
+	static class TestReporter2 extends TestReporter {
```

Comment:
This change was necessary since the new tests actually instantiate the reporter, whereas the exiting ones are only checking the class property in the configuration.